packages/react-button/src/components/Button/useButtonClasses.ts

```diff
@@ -114,6 +114,23 @@ export const useButtonClasses = makeVariantClasses<ButtonState>({
           marginLeft: 'var(--button-contentGap)',
         },
 
+        '@media (forced-colors: active)': {
```

Comment:
we have `EdgeChromiumHighContrastSelector` from styling package. I think we should use that. although this me realized, I don't think we need  to specify `screen` for this query? I am not completely sure

Reply:
Done